tools/README.template.md

```
@@ -110,8 +125,9 @@ We're committed to making your experience with Determinate Nix and FlakeHub as s
 [dependabot]: https://github.com/dependabot
 [det-nix]: https://docs.determinate.systems/determinate-nix
 [determinate]: https://docs.determinate.systems
-[detsys]: https://determinate.systems/
+[detsys]: https://determinate.systems
```

markdownlint-cli2 (0.22.1)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "detsys"

(MD053, link-image-reference-definitions)